aosp_diff/celadon_ivi/frameworks/av/cmds/stagefright/0002-Enable-stagefright-and-recorder-for-debug.patch

```diff
@@ -0,0 +1,528 @@
+From 95416df19255fa73a2990a8606716b00c73d12fb Mon Sep 17 00:00:00 2001
+From: "Martin, Chen" <haochuan.z.chen@intel.com>
+Date: Mon, 3 Jul 2023 07:01:02 -0400
+Subject: [PATCH] Enable stagefright and recorder for debug
+
+Tracked-On: OAM-111014
+Signed-off-by: Martin, Chen <haochuan.z.chen@intel.com>
+---
+ cmds/stagefright/Android.mk                   |  27 +++
+ cmds/stagefright/AudioPlayer.cpp              |   5 +-
+ cmds/stagefright/SineSource.cpp               |  71 +++++-
+ cmds/stagefright/SineSource.h                 |   4 +-
+ cmds/stagefright/WavRecorder.cpp              | 219 ++++++++++++++++++
+ cmds/stagefright/WaveWriter.h                 |   9 +-
+ cmds/stagefright/stagefright.cpp              |  28 ++-
+ .../media/stagefright/foundation/MediaDefs.h  |   1 +
+ 8 files changed, 346 insertions(+), 18 deletions(-)
+ create mode 100644 cmds/stagefright/WavRecorder.cpp
+
+diff --git a/cmds/stagefright/Android.mk b/cmds/stagefright/Android.mk
+index 803c4a477a..857a0808e4 100644
+--- a/cmds/stagefright/Android.mk
++++ b/cmds/stagefright/Android.mk
+@@ -274,3 +274,30 @@ LOCAL_LICENSE_CONDITIONS:= notice
+ LOCAL_NOTICE_FILE:= $(LOCAL_PATH)/NOTICE
+ 
+ include $(BUILD_EXECUTABLE)
++#################################################################################
```

Comment:
As this introduces new unit test code, lets create new project and maintain it there. any changes to existing aosp code is ok to maintain as diff but introduction of new library/binary needs new project

Reply:
update